extension/chrome/settings/index.ts

```diff
@@ -309,6 +311,38 @@ Catch.try(async () => {
     }));
   };
 
+  const renderBlogPostList = async () => {
+    const errorRetrievingPosts = () => {
+      Xss.sanitizeAppend('.blog_post_list', `<div class="line>Error retrieving blog posts...try again later.</div>\n`);
+    };
+    try {
+      const res = await Api.ajax({
+        url: 'https://flowcrypt.com/feed',
+        method: 'GET',
+        dataType: 'json',
+        async: true
+      }, Catch.stackTrace());
+
+      interface BlogPostData {
+        url: string;
+        title: string;
+        date: string;
+      }
+      if (res instanceof Array) {
+        (res as BlogPostData[]).map(post => {
+          $('.blog_post_list').append(
+            `<div class="line"><a href="https://flowcrypt.com${Xss.escape(post.url)}" target="_blank">${Xss.escape(post.title.trim())}</a> (${Xss.escape(post.date.trim())})</div> \n`);
+        });
+      } else {
```

Comment:
All of the lines from here down will end up being redundant. Actually, because it's just blog post, you don't even need to render errors here - if the network is down, you can just skip displaying posts - they are not that important to always show. But if you wanted to handle them, maybe a better place would be in the new `.catch()` block above, because then you only have to write it one time, and don't need to have a method for it.